### DISCUSSION TITLE
SV06-Plus - problems to connect to printer BTT Pad 7

### DISCUSSION THREAD

**Discussion by Manfredo1952** [bassamanator/Sovol-SV06-firmware#144] [Q&A]
After a few weeks of inactivity I started the SV06+ today and the connected BTT-Pad7 - Printer responded well and I could home all axis.  They I saw that all of the a stuff (mainsail etc) was up for updates, i.e. I hit the button "update all" - to bad I dod that. After the updates, no more connection possible. Neither via BTT-Pad nor via WebBrowser, I can log into mainsail's webpage, but it dioes report this error . I have then tried to flash again the Klipper FW, nothing. I have recompiled the FW. same.    What else can I do, please    -  report as follows    

Klipper reports: ERROR

MCU Protocol error

This is frequently caused by running an older version of the
firmware on the MCU(s). Fix by recompiling and flashing the
firmware.

Your Klipper version is: v0.12.0-102-g9f41f53c
MCU(s) which should be updated:
mcu: Current version v0.12.0-10-gea2f6bc0
CB1: Current version v0.11.0-122-ge6ef48cd
Up-to-date MCU(s):
<none>

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.

mcu 'CB1': Command format mismatch: query_adxl345 oid=%c rest_ticks=%u vs query_adxl345 oid=%c clock=%u rest_ticks=%u

++

KIAUH reports as follows

<img width="410" alt="grafik" src="https://github.com/bassamanator/Sovol-SV06-firmware/assets/56330568/22f14793-3fae-4c7d-8b96-b13d6cb090be">

Thanks

**Comment by tavalin** [comment]
You need to update/flash the klipper on the MCU to match the version running on the BTT Pad7

**Comment by Manfredo1952** [comment]
Found the solution when I checked the BTT FB Group - there are several users having that same problem after updating. Solution is here   https://www.youtube.com/watch?v=cnthbR3tS9M
